instrumentation/github.com/gin-gonic/gin/otelgin/option.go

```diff
@@ -70,6 +77,18 @@ func WithFilter(f ...Filter) Option {
 	})
```

Comment:
I can't suggest an edit here, and it seems you missed my last comment.
In this method's comment, replace `All filters` with `All gin and net/http filters`